src/GeneralStateTestsFiller/stStaticCall/static_call_value_inherit_from_callFiller.json

```diff
@@ -0,0 +1,76 @@
+{
+    "static_call_value_inherit_from_call" : {
+        "env" : {
+            "currentCoinbase" : "2adc25665018aa1fe0e6bc666dac8fc2697ff9ba",
+            "currentDifficulty" : "0x020000",
+            "currentGasLimit" : "10000000",
+            "currentNumber" : "1",
+            "currentTimestamp" : "1000",
+            "previousHash" : "5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6"
+        },
+        "expect" : [
+            {
+                "indexes" : {
+                    "data" : -1,
+                    "gas" : -1,
+                    "value" : -1
+                },
+                "network" : ["Metropolis"],
+                "result" : {
+                    "094f5374fce5edbc8e2a8697c15331677e6ebf0b" : {
+                        "storage" : {
+                            "0x0" : "0x1",
+                            "0x1" : "0x0"
+                        }
+                    }
+                }
+            }
+        ],
+        "pre" :
+        {
+            "a94f5374fce5edbc8e2a8697c15331677e6ebf0b" : {
+                "balance" : "1000000000000",
+                "code" : "",
+                "nonce" : "0",
+                "storage" : {
+                }
+            },
+            "094f5374fce5edbc8e2a8697c15331677e6ebf0b" : {
+                "balance" : "1",
+                "code" : "{ [[0]] (STATICCALL 50000 0xc94f5374fce5edbc8e2a8697c15331677e6ebf0b 0 0 0 32) [[1]] (MLOAD 0) }",
+                "nonce" : "0",
+                "storage" : {
+                    "0x1": "1"
+                }
+            },
+            "c8b90f8d999d9d9988209481999a9889098a89a0" : {
+                "balance" : "0",
+                "code" : "{ (CALL 100000 0x094f5374fce5edbc8e2a8697c15331677e6ebf0b 10 0 0 0 0) }",
+                "nonce" : "0",
+                "storage" : {}
+            },
+            "c94f5374fce5edbc8e2a8697c15331677e6ebf0b" : {
+                "balance" : "1",
+                "code" : "{ (MSTORE 0 (CALLVALUE)) (RETURN 0 32) }",
```

Comment:
The return value will be returned to "094f5374fce5edbc8e2a8697c15331677e6ebf0b".